lib/activity_reporter.rb

```diff
@@ -0,0 +1,43 @@
+# generates text for a report on druid count and logs
+class ActivityReporter
+  DRUID_REGEXP = '[[:lower:]]{2}\\d{3}[[:lower:]]{2}\\d{4}'.freeze
+
+  def output(log_files=default_log_files)
+    today = Time.now.to_date.to_s
+    counter = {}
+    puts "*" * 20
+    log_files.each do |file|
+      unless File.exist?(file) && File.size(file) > 0
+        puts "EMPTY or NON-EXISTENT: #{file}"
+        next
+      end
+      extract_druid(file, today, counter)
+      druid_counts = counter.keys.size
+      if druid_counts.zero?
+        puts "No activity #{today}, DRUID count: #{druid_counts}\n"
```

Comment:
isn't "No activity" misleading if there are errors?   What do you think of "No druids processed" or just let it be druid_count 0?   ask @blalbrit ?